src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -125,7 +126,37 @@ def create_launcher(fluent_launch_mode: LaunchMode = None, **kwargs):
         return SlurmLauncher(**kwargs)
 
 
+def deprecate(old_arg, new_arg, converter):
```

Comment:
This is a generic method that can be used to deprecate other arguments as well in future.

Reply:
This function should be extracted from this module for reuse. We can embed the generalised warning within this function. (Almost) all warnings will have the same structure.